### PR TITLE
Add some documentation on how to request model output

### PR DESCRIPTION
**Pull request by bartgol**
Added in the `docs/mkdocs/docs/user` folder.

### CODE REVIEW

**Review by AaronDonahue — APPROVED** [review]
@bartgol , this is fantastic!  Thank you for putting all of this into place, and filling out some sections already.  I was going to comment on minor edits but I see this PR as a first step and presumably we will iterate on the text as we fill out more of the sections.
I'm looking forward to adding documentation and fleshing these sections out.

**Comment by bartgol** [comment]
Interesting: this branch was originally on top of `bartgol/docs-work`. When that branch got merged, github claimed that "The merge-base changed after approval.", so it dismissed Aaron's approval. More importantly, it didn't seem to recognize that the first two commits were now in master, and, instead of removing them from this PR, it kept them there. Not knowing whether the two commits would have been duplicated upon merge, and since a new approval was now necessary, I decided to rebase on master.

@AaronDonahue can you please re-approve?
gh

**Review by AaronDonahue — APPROVED** [review]
re-approved

**Pull request merged**
